src/main/java/hudson/plugins/tics/TicsPublisher.java

```diff
@@ -128,7 +130,7 @@ public void perform(@Nonnull final Run<?, ?> run, @Nonnull final FilePath worksp
             gateData = null;
         }
 
-        run.addAction(new TicsPublisherBuildAction(run, tqiData, gateData, tiobeWebBaseUrl));
+        run.addAction(new TicsPublisherBuildAction(run, ticsPath1, tqiData, gateData, tiobeWebBaseUrl));
```

Comment:
Thy is ticsPath1 passed? Can this varable be named more specific?

Reply:
The variable already existed and is named 'ticsPath1'. This is already specific I would say: it is a TICS path, as opposed to a normal file system path. It has the suffix 1 to prevent hiding a field named 'ticsPath'. But again, the variable already exist. I want to keep the change set minimal.